stmhal/usbd_conf.c

```diff
@@ -96,9 +96,51 @@ void HAL_PCD_MspInit(PCD_HandleTypeDef *hpcd)
     HAL_NVIC_EnableIRQ(OTG_FS_IRQn);
   } 
   
-  #if defined(USE_USB_HS)
   else if(hpcd->Instance == USB_OTG_HS)
   {
+#if defined(USE_USB_HS_IN_FS)
+
+    /* Configure USB FS GPIOs */
+    __GPIOB_CLK_ENABLE();
+
+    /* Configure DM DP Pins */
+    GPIO_InitStruct.Pin = (GPIO_PIN_14 | GPIO_PIN_15);
+    GPIO_InitStruct.Mode = GPIO_MODE_AF_PP;
+    GPIO_InitStruct.Pull = GPIO_NOPULL;
+    GPIO_InitStruct.Speed = GPIO_SPEED_HIGH;
+    GPIO_InitStruct.Alternate = GPIO_AF12_OTG_HS_FS;
+    HAL_GPIO_Init(GPIOB, &GPIO_InitStruct);
+
+#if defined(MICROPY_HW_USB_VBUS_DETECT_PIN)
+    /* Configure VBUS Pin */

```

Comment:
Sure no problem. If I make the update later to the common code  stmhal/usbd_conf.c I think it works for the STM32F429I as well.